spec/shr_foundation_map_stu3.txt

```diff
@@ -0,0 +1,103 @@
+Grammar:	Map 5.1
+Namespace:	shr.core
+Target:		FHIR_STU_3
+
+
+DomainResource maps to DomainResource:
+//	_Entry.EntryId maps to id
+	Language maps to language
+	Narrative maps to text
+	ImplicitRules maps to implicitRules
+	Metadata.VersionId maps to meta.versionId
+//	Metadata.SourceSystem maps to meta.extension 
+	// in R4, this will be meta.source
+	// _Entry.EntryType maps to meta.profile // (this will force a profile -- don't do it)
+	Metadata.LastUpdated maps to meta.lastUpdated
+	Metadata.SecurityLabel maps to meta.security
+	Metadata.Tag maps to meta.tag
+
+Composition maps to Composition:
+	Identifier maps to identifier
+	Status maps to status
+	Type maps to type
+    Section maps to section
+    Section.DomainResource maps to section.entry  (slice on = reference.resolve(); slice on type = profile; slice strategy = includes)
+	Section.Section maps to section.section
+
+Narrative maps to Narrative:
+	NarrativeText maps to div
+	NarrativeQualifier maps to status
+
+
+ValueSet maps to ValueSet:
```

Comment:
More of an FYI, but this line's reference to "_ValueSet:_" in `ValueSet maps to ValueSet:` clashes with our CIMPL keyword **ValueSet:**